contracts/DebtToken.sol

```diff
@@ -184,7 +195,7 @@ contract DebtToken is Ownable {
   calculate the total number of passed interest cycles and coin value
   */
   function calculateInterestDue() public constant returns(uint256 _coins,uint256 _cycle){
-    if(!isTermOver())
+    if(!isTermOver() || !isLoanFunded())
```

Comment:
` || !isLoanFunded()` is already implicit in `isTermOver()`

Reply:
This isn't actually redundant. Removing it causes `Should confirm loanValue does not increase after refundLoan` test to fail.

Reply:
will check that out